### ISSUE TITLE
Wrong error on `isinstance(..., Literal[...])`

### ISSUE THREAD

**Issue by JelleZijlstra** [microsoft/pyright#2772]
This code:
```python
from typing import Literal
def f(i: int):
    if isinstance(i, Literal[1]):
        pass
```
produces (with Pylance 2021.12.2) an error `Unnecessary isinstance call; "int" is always an instance of "Literal[1]"`. That error is nonsense; it should instead say that `Literal[1]` is not a supported argument to `isinstance()`.

**Comment by erictraut** [comment]
Thanks for the bug report. This will be addressed in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.202, which I just published. It will also be included in the next release of pylance.
